### PR TITLE
[DOCS] Adds note on how to update custom rules of Metric anomaly jobs

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR adds a note to the `Inspect metric anomalies` page about how to update the custom rules of a pre-configured Metric anomaly detection job.

### Preview

[Inspect metric anomalies](https://observability-docs_249.docs-preview.app.elstc.co/guide/en/observability/master/inspect-metric-anomalies.html)

### CODE REVIEW

**Comment by szabosteve** [comment]
@elasticmachine test this please

File: docs/en/observability/inspect-metric-anomalies.asciidoc [hunk 1]
```diff
@@ -31,21 +35,34 @@ Machine learning jobs analyze the last four weeks of data and continue to run in
...
-6. You're now ready to explore your metric anomalies. Click *View anomalies* to view the
-{ml-docs}/ml-gs-results.html[Anomaly Explorer in Machine Learning].
+6. You're now ready to explore your metric anomalies. Click *View anomalies* to 
+view the {ml-docs}/ml-gs-results.html[Anomaly Explorer in Machine Learning].
+
+[NOTE]
+=====
+These pre-defined {anomaly-jobs} use {ml-docs}/ml-rules.html[custom rules]. You 
+can update the rules in the *Anomaly explorer* by selecting the *Configure 
+rules* option under *actions*. The changes take effect for new results only. If
```
Comment (EamonnTP) [inline docs/en/observability/inspect-metric-anomalies.asciidoc:51]:
```suggestion
These pre-defined {anomaly-jobs} use {ml-docs}/ml-rules.html[custom rules]. To
update the rules in the *Anomaly Explorer*, select *actions > Configure 
rules*. The changes only take effect for new results.
```

**Review by EamonnTP — CHANGES_REQUESTED** [review]
Looks good! Just a minor suggestion.

**Comment by szabosteve** [comment]
Thanks @EamonnTP, I addressed your comment via 996cf49

File: docs/en/observability/inspect-metric-anomalies.asciidoc [hunk 2]
```diff
@@ -31,21 +35,34 @@ Machine learning jobs analyze the last four weeks of data and continue to run in
...
-{ml-docs}/ml-gs-results.html[Anomaly Explorer in Machine Learning].
+6. You're now ready to explore your metric anomalies. Click *View anomalies* to 
+view the {ml-docs}/ml-gs-results.html[Anomaly Explorer in Machine Learning].
+
+[NOTE]
+=====
+These pre-defined {anomaly-jobs} use {ml-docs}/ml-rules.html[custom rules]. To
+update the rules in the *Anomaly Explorer*, select *actions > Configure 
+rules*. The changes only take effect for new results.
+you want to apply the changes to existing results, clone and rerun the job.
```
Comment (EamonnTP) [inline docs/en/observability/inspect-metric-anomalies.asciidoc:52]:
```suggestion
If you want to apply the changes to existing results, clone and rerun the job.
```

**Comment by szabosteve** [comment]
@EamonnTP Thanks!

**Review by EamonnTP — APPROVED** [review]

**Pull request merged**